cypress/e2e/shared/loadDataSources.test.ts

```diff
@@ -175,8 +174,6 @@ describe('Load Data Sources', () => {
     // use new data explorer, since old data explorer is being deprecated
     cy.getByTestID('nav-item-data-explorer').click({force: true})
 
-    cy.getByTestID('script-query-builder-toggle').click()
```

Comment:
Minor adjustments in this test, since the new data explorer is now the default.

Reply:
Adding it back since we still want to set the default value as `scriptQueryBuilder: false` in `src/shared/reducers/app.ts`

Reply:
We undid this changed, based on the convo documented above.